man/mono.1

```diff
@@ -1652,11 +1652,11 @@ Automatically generates sequence points where the
 IL stack is empty.  These are places where the debugger can set a
 breakpoint.
 .TP
-\fBgen-compact-seq-points\fR
```

Comment:
And now that I look at it, the same likely applies to `gen-seq-points-file` as well which was removed in https://github.com/mono/mono/commit/eb8cc5ca57c7fd03ca0026c9c0bcef9734c78776. 

It's used in monodroid here: https://github.com/xamarin/xamarin-android/blob/14260f43fc47abc9aee0ff5a0ca10a10ac2e3384/src/Xamarin.Android.Build.Tasks/Tasks/Aot.cs#L329